src/test/java/com/wikia/webdriver/elements/communities/desktop/components/navigation/local/CommunityHeaderDesktop.java

```diff
@@ -146,7 +146,7 @@ public CommunityHeaderDesktop openExploreMenu() {
 
   public SpecialWikiActivityPageObject clickExploreWikiActivityLink() {
     wait.forElementClickable(exploreWikiActivityLink);
-    scrollAndClick(exploreWikiActivityLink);
```

Comment:
why are we replacing scrollAndClick with just click?

Reply:
Sometimes page was scrolled before clicking the exploreTab and that was the reason test failed. So I've changed before for (https://github.com/Wikia/selenium-tests/pull/2513/files) `scrollTo(exploreTab)` but it worked only locally. Now it should work as scrolling to exploreTab and just choosing the option e.g. Videos